server/src/entity/InventoryItem.ts

```diff
@@ -21,34 +30,42 @@ export enum itemCategory {
 @Entity()
 export class InventoryItem extends BaseEntity {
   @PrimaryColumn()
+  @IsString()
   public name: string;
 
   @Column({
     default: 0,
   })
+  @IsNumber()
   public quantity: number;
 
   @Column({
     type: 'enum',
     enum: itemCategory,
     default: itemCategory.UNKNOWN,
   })
+  @IsEnum(itemCategory)
   public category: itemCategory;
 
   @Column({ length: 50, default: 'default_logo.jpg', nullable: true })
+  @IsString()
   public logoPath: string;
 
   @Column({ type: 'simple-array' })
+  @IsArray()
```

Comment:
is there any way you can specify the type of the array?

Reply:
I tried. Cannot find it